unibuild/utility/qt.py

```diff
@@ -44,19 +54,33 @@ def get_qt_install(qt_version, qt_version_minor, vc_version):
             if os.path.isfile(f):
                 return os.path.realpath(p)
     except Exception:
-        res = None
+        pass
 
-    # We should try the custom Qt install path as well
-    if res is None:
-        try:
-            p = os.path.join(config['qt_CustomInstallPath'], "{}".format(qt_version + "." + qt_version_minor
-                                                                         if qt_version_minor != '' else qt_version),
+    # Look in the path
+    try:
+        f = shutil.which("qmake.exe")
+        if f is not None and os.path.isfile(f):
+            return os.path.realpath(f)
+    except Exception:
+        pass
+
+    # Look for qtcreator.exe, which is in %qt%/Tools/QtCreator/bin, then walk
+    # back up to the root from there
+    try:
+        f = shutil.which("qtcreator.exe")
+        if f is not None and os.path.isfile(f):
+            root = os.path.join(os.path.dirname(f), "..", "..", "..")
+
+            p = os.path.join(root, "{}".format(qt_version + "." + qt_version_minor
+                                               if qt_version_minor != '' else qt_version),
                              "msvc{0}_64".format(vc_year(vc_version)))
             f = os.path.join(p, "bin", "qmake.exe")
             if os.path.isfile(f):
                 return os.path.realpath(p)
-        except Exception:
-            res = None
+    except Exception:
```

Comment:
Catching everything and then passing is very bad practice and makes debugging very hard.

Reply:
Well, this is an internal script and it was already doing that. It's assuming any error means Qt isn't there and moves on to the next step.

I removed the useless `res`, but I ain't rewriting this thing.

Reply:
I mean it should be fine in this context but normally you never want to do that.